saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/modern.scss

```diff
@@ -1040,6 +1071,108 @@ form.search {
 		
 }
 
+.judgment-info {
```

Comment:
I wonder if the info sections css wouldn't be more generic, not judgment specific? The same way as the js handling it.